src/CMakeLists.txt

```diff
@@ -123,6 +123,11 @@ if(HAVE_INTEL)
     HAVE_BETTER_YASM_ELF64)
 endif()
 
+if(HAVE_ARM)
+  if(HAVE_ARMV8_SIMD)
```

Comment:
i think `HAVE_ARMV8_SIMD` implies `HAVE_ARM`. so `if(HAVE_ARM)` is not necessary. am i right? and why bother defining `HAVE_AARCH64_ASM_ELF64`? just for the sake of symmetry ?

Reply:
yes, you are right, HAVE_AARCH64_ASM_ELF64 is not nessary, just for corresponding to HAVE_BETTER_YASM_ELF64. we can just use HAVE_ARMV8_SIMD.